### DISCUSSION TITLE
Quanto vocês cobram/cobrariam pelo seguinte serviço

### DISCUSSION THREAD

**Discussion by Luleps** [frontendbr/forum#1651] [Discussão]
Olá, o que quero propôr aqui é uma discussão onde possamos falar abertamente de valores mesmo, sempre que procurei sobre o assunto as respostas foram superficiais e até então não faço idéia de como me portar diante de algumas situações.

Bom, atualmente presto serviço remoto de casa para duas empresas, cuido das mídias dessas empresas montando (literalmente) posts para Instagram e Facebook, faço a arte no Photoshop e penso no texto. E cuido também do site da empresa que eu mesmo criei no WordPress e em breve farei anúncios no Google também.

Pois bem, as vezes é bem trabalhoso e as vezes é bem tranquilo e essas empresas juntas me pagam R$1500 por mês.

Minha dúvida e a discussão que quero abrir é, recentemente uma empresa do norte do Brasil me pediu os mesmos serviços, quanto devo cobrar? O que cobro atualmente é justo levando em consideração o fato de ser remoto? Na mesma situação quanto vocês cobrariam? Tanto da empresa nordestina quanto das q atualmente já presto serviço. Espero mesmo obter respostas que me ajude, mesmo prestando um serviço "simples".

Obrigada.

**Comment by eduardosoareszup** [comment]
Acredito que a melhor forma de cobrar é você precificar sua hora para cada tipo de serviço e calcular quanto que cada empresa está te demandando. 

Vamos supor que para cada post com imagem personalizada você demore cerca de 4 horas para criar a arte, montar o texto, enviar para o cliente e ele aprovar. Você pode por um determinado valor para hora desse tipo de serviço, digamos R$ 25. Para cada serviço desse seria em torno de R$ 100, você faz 3 post por semana, então no mês para esse tipo de serviço seria em torno de R$ 900.

Outro serviço talvez seja responder comentários nas redes sociais, ai vai depender do tamanho das páginas, digamos que você invista 6 hora por semana na página, você pode cobrar por exemplo R$ 15 a hora desse tipo de trabalho, por ele exigir menor conhecimento técnico e talvez ser mais simples. E por ai vai.

A linha principal que vejo a galera cobrando é dessa forma, eles meio que criam um plano para cada cliente, planejando quantas horas eles devem gastar para cada tarefa e determinam um valor para cada tipo de tarefa. Quanto mais complexa mais caro e quanto mais simples mais barata. No final eles multiplicam e somam tudo e acabam tendo uma estimativa de quanto vai custar.

Claro que cada cliente vai ter uma demanda, alguns meses vão ser mais tranquilos e outros mais puxados, então a ideia é você colocar na média e tentar sempre ter uma receita recorrente.

Espero ter ajudado 😄 .

**Observação**: Os valores que coloquei são apenas para exemplo, a melhor pessoa para precificar pelo serviço é você. Mas caso queira ter um norte você pode ir pesquisando em site de freelancers quanto que o pessoal ta cobrando por hora para cada tipo de serviço.

**Comment by kavalcante** [comment]
Cara, essas coisas variam muito com base no seu nível de conhecimento mesmo. Por isso é díficil alguém dar uma resposta que não seja muito genérico.

Eu recomendo você chegar em uma estimativa por hora que seja justa pra você. Define um número que é bom o suficiente.

Um problema de trampar sempre com freela é que pode ser que seus clientes parem de ser seus clientes, isso se chama churn, toda empresa tem. Mas quando se é funcionário, isso impacta menos a sua vida.

Cobre um valor que dê pra você guardar dinheiro, pra quando um cliente sair, você não tenha que reduzir drasticamente seus custos do dia para a noite.

Um exemplo, se você precisa de R$ 2.000,00 pra viver e você cobra R$ 20 a sua hora, se você trabalhar 8h por dia, você vai tirar no máximo R$ 3.520,00 se tiver todos os horários cheios. Isso é bom, pq você tem R$ 1.200,00 de "margem" de erro. Ou seja, você pode ficar teoricamente com 60 horas a menos de trabalho por mês, que você ainda vai ganhar os seus R$ 2.000,00.

Enfim, o exemplo acima não considera o valor que é cobrado em média por esse serviço e essa informação varia muito e eu nem trabalho com isso pra saber também. Mas é uma forma de você olhar também.

Outra é cobrar por cliente mesmo (os que dão mais trampo tu cobra um preço e outros tu cobra outro). Com o tempo e a experiência, você vai aprender a precificar melhor o que você acha justo. Não se importe muito em errar, pq todo mundo erra no começo. Depois é só arrumar.

**Comment by kivervinicius** [comment]
Apenas acrescentando o que os colegas já disseram, conclui-se que um calculo pelo menos que superficial e mediano de valor/hora é o necessário, outra coisa também é verificar quanto de imposto você paga (isso que se você paga, caso não seja MEI, recomendo extremamente), eu não sei qual o grau de trabalho que vc tem no maior mes do ano por exemplo e quanto que vc menos trabalha no ano, tenta fazer um calculo seguindo isso tbm como parâmetro. Pelo valor que vc passa eu acredito ser um bom valor atualmente caso a empresa do norte possua o mesmo grau de dificuldade dos outros, se não somente média/hora mesmo.

ps: Dica faça contratos no mínimo trimestrais e para ter um tipo de segurança.

**Comment by lucasinocente** [comment]
Acrescentando, é interessante ter um cálculo de custo mensal pra exercer a profissão também, por ex:

Internet = R$ 200
aluguel / coworking = R$ 500
luz = 100
depreciação da máquina por mês, pensando que um computador de 4k dure 2 anos = 200
depreciação móveis de escritório / ar condicionado = 200
imposto = 10% (de 2k = R$ 200)
contador = R$ 200

Só aí já temos um custo de trabalho de uns R$ 1600

**Comment by tarciozemel** [comment]
Os colegas levantaram pontos e técnicas de precificação bem importantes para serem levados em conta, mas os valores de mercado, "efeitos psicológicos" relacionados a valores, auto-valorização etc. também podem ser considerados.

A princípio, parece que você realmente está cobrando baratinho (considerando que faça um bom serviço).
